spec/autofill-spec.ts

```diff
@@ -25,4 +26,22 @@ describe('autofill', () => {
     const value = await w.webContents.executeJavaScript("document.querySelector('input').value");
     expect(value).to.equal('Eric Clapton');
   });
+
+  it('can be selected via keyboard for a <datalist> with time type', async () => {
+    const w = new BrowserWindow({ show: true });
+    await w.loadFile(path.join(fixturesPath, 'pages', 'datalist-time.html'));
+
+    const inputText = '11P'; // 1:01 PM
+    for (const keyCode of inputText) {
+      w.webContents.sendInputEvent({ type: 'keyDown', keyCode: 'Tab' });
+      w.webContents.sendInputEvent({ type: 'keyDown', keyCode });
```

Comment:
@codebytere why the extra keyDowns here?

Reply:
They were necessary in local testing for consistent success - it flaked a bit without them